manifest.json

```diff
@@ -6,7 +6,7 @@
         {
           "platforms": [
             {
-              "dockerfile": "src/alpine/3.6",
+              "dockerfile": "src/alpine/3.6/amd64/base",
```

Comment:
I would like to see the amd64 directory as the last folder in the path as that aligns with the tagging strategy.

Reply:
Why was base introduced?  The convention is for the folder structure to align with the tag name.

Reply:
This was to distinguish between the `amd64/ForHelix` and `amd64/WithNode` directories.

Will remove; however it leaves the directory structure imo a little funky.

```
alpine
   |
   ----3.6
            |
            ----Dockerfile
            |
            ----WithNode
                         |
                         -----Dockerfile
            ----ForHelix
                         |
                         -----Dockerfile

```

If that is how you expect it then I am fine with it.

Reply:
Will make the change as suggested, let me know if you want it changed back.